src/util.rs

```diff
@@ -121,7 +121,7 @@ pub(crate) fn check_policy_stage<T: Serialize, U: Serialize>(
                 for violation in violations {
                     errors.push(PolicyViolation {
                         provenance: policy_file.to_owned(),
-                        violation,
+                        violation: Box::new(violation),
```

Comment:
Why is this boxed?

Reply:
same as https://github.com/open-telemetry/weaver/pull/943#discussion_r2380285736